### PR TITLE
feat(documentation): adds sizing documentation

### PR DESCRIPTION
**Pull request by b1aserlu**

### CODE REVIEW

File: packages/documentation/src/stories/utilities/sizing/sizing.stories.ts [hunk 1]
```diff
@@ -1,15 +1,122 @@
-import { Meta, StoryObj } from '@storybook/web-components';
+import type { Args, Meta, StoryObj, StoryContext } from '@storybook/web-components';
+import { html } from 'lit/static-html.js';
```
Comment (imagoiq) [inline packages/documentation/src/stories/utilities/sizing/sizing.stories.ts:2]:
```suggestion
import { html } from 'lit';
```
Doesn't make a big difference, but it's more future-proof.

File: packages/documentation/src/stories/utilities/sizing/sizing.stories.ts [hunk 2]
```diff
@@ -1,15 +1,122 @@
...
+        <div class="flex-fill">
+          <div class="${classes}"></div>
+        </div>
+
+        <ul class="legend list-unstyled">
+          <li class="d-flex align-items-center">
+            <div class="h-regular w-regular me-mini content"></div>
+            <span>content</span>
+          </li>
+        </ul>
```
Comment (imagoiq) [inline packages/documentation/src/stories/utilities/sizing/sizing.stories.ts:101]: Can you explain why it's needed? It confuses me.

**Review by imagoiq — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx [hunk 3]
```diff
@@ -1,11 +1,68 @@
...
 <Meta of={SizingStories} />
 
 # Sizing
 
-<div className="alert alert-info">
-  <h2 className="alert-heading">TODO:</h2>
-  <p>Document differences to bootstrap and link over for the rest.</p>
+## Height & Width
+
+Height and width can be set using `h-*` and `w-*` prefixes, max-height and max-width can be set using prefixes mh-_ and mw-_ .
```
Comment (imagoiq) [inline packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:12]:
```suggestion
Height and width can be set using `h-*` and `w-*` prefixes, max-height and max-width can be set using prefixes `mh-*` and `mw-*` .
```

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx [hunk 4]
```diff
@@ -1,11 +1,68 @@
...
+></Source>
+
+#### Example
+
+<Canvas of={SizingStories.Sizes} />
+<div className="hide-col-default">
+  <Controls of={SizingStories.Sizes} />
+</div>
+
+<StylesPackageImport components={['sizing']} required={{}} />
```
Comment (imagoiq) [inline packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:68]:
```suggestion
```
It's not a component and we don't display it on other similar docs pages like spacing.

File: packages/documentation/src/stories/utilities/sizing/sizing.stories.ts [hunk 5]
```diff
@@ -1,15 +1,101 @@
...
   },
 };
 
 export default meta;
 
 type Story = StoryObj;
 
-export const Default: Story = {};
+function renderSizing(args: Args) {
+  const classes = `content h-${args.height} w-${args.width} mh-${args.maxHeight} mw-${args.maxWidth}`;
```
Comment (imagoiq) [inline packages/documentation/src/stories/utilities/sizing/sizing.stories.ts:73]: `args.maxHeight` and `args.maxWidth` can be undefined, resulting in adding class like `mh-undefined`

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx [hunk 6]
```diff
@@ -1,11 +1,68 @@
-import { Meta } from '@storybook/blocks';
+import { Canvas, Controls, Meta, Source } from '@storybook/blocks';
+import { formatAsMap } from '../../../utils/sass-export';
 import * as SizingStories from './sizing.stories';
+import StylesPackageImport from '../../../shared/styles-package-import.mdx';
 
 <Meta of={SizingStories} />
 
 # Sizing
```
Comment (imagoiq) [inline packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:9]: Here we are missing a `.lead` text that introduces the page.

**Review by imagoiq — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx [hunk 7]
```diff
@@ -1,11 +1,68 @@
...
+### Parent Based
+
+Bootstrap offers sizing classes with the suffixes bellow. They allow you to set a height / width as a percentage of the parent element (see [Bootstrap Sizing](https://getbootstrap.com/docs/5.3/utilities/sizing/) for reference).
+
+> - `*-25`
+> - `*-50?`
+> - `*-75`
+> - `*-100`
+
+#### Example
```
[the remark this answers is not in the document]
Comment (b1aserlu) [reply packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:32]: No, this is only the example for the parent-based sizing the next section has its own examples

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx [hunk 8]
```diff
@@ -1,11 +1,71 @@
...
 
 <Meta of={SizingStories} />
 
 # Sizing
 
-<div className="alert alert-info">
-  <h2 className="alert-heading">TODO:</h2>
-  <p>Document differences to bootstrap and link over for the rest.</p>
+<div className="lead">
+  Sizing utitlity classes are used to influence the height and width of elements to which they are
```
Comment (imagoiq) [inline packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:11]:
```suggestion
  Sizing utility classes are used to influence the height and width of elements to which they are
```

**Review by imagoiq — APPROVED** [review]

File: packages/documentation/src/stories/utilities/sizing/sizing.stories.ts [hunk 9]
```diff
@@ -1,15 +1,95 @@
...
+      name: 'max-height',
+      description: 'Set the maximum height of the cube',
+      control: {
+        type: 'select',
+      },
+      options: ['none', ...sizingOptions],
+    },
+    maxWidth: {
+      name: 'max-width',
+      description: 'Set the maximum width of the cube',
```
Comment (b1aserlu) [inline packages/documentation/src/stories/utilities/sizing/sizing.stories.ts:52]:
```suggestion
      description: 'Set the height of the rectangle',
      control: {
        type: 'select',
      },
      options: sizingOptions,
    },
    width: {
      name: 'width',
      description: 'Set the width of the rectangle',
      control: {
        type: 'select',
      },
      options: sizingOptions,
    },
    maxHeight: {
      name: 'max-height',
      description: 'Set the maximum height of the rectangle',
      control: {
        type: 'select',
      },
      options: ['none', ...sizingOptions],
    },
    maxWidth: {
      name: 'max-width',
      description: 'Set the maximum width of the rectangle',
```

**Pull request merged**